ts/screens/modal/IdentificationModal.tsx

```diff
@@ -97,6 +97,15 @@ const styles = StyleSheet.create({
   header: {
     fontSize: 20,
     lineHeight: 22
+  },
+  bottomContainer: {
+    position: "absolute",
+    bottom: 0,
```

Comment:
Maybe i would increase a bit more the bottom value
```suggestion
    bottom: 20,
```

this is what i see on an Android phone

### Before 
<img src="https://user-images.githubusercontent.com/3959405/117126347-ec8a7d80-ad9a-11eb-8a69-a398eec639b7.png" height="550"/>

### After
<img src="https://user-images.githubusercontent.com/3959405/117126446-0b890f80-ad9b-11eb-91e2-ce27529f4455.png" height="550"/>